src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -1066,7 +1066,8 @@ private static String forAgency(final String agencyId) {
     }
 
     public InmateDetail getOffender(final String offenderNo) {
-        return  offenderRepository.findOffenderByNomsId(offenderNo)
+        telemetryClient.trackEvent("getOffender 1.1_beta version used");
```

Comment:
I don't think this method is called, but since it is determined by a header, log in app insights if it is